css-pseudo-classes/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>css pseudo classes</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <h1>Hover Button</h1>
+    <input type="button" id="button" value="I'm a button!">
```

Comment:
Since we're not working with forms, this should be a `button`. You will only use `input` elements when you're working with a `form`.

Reply:
<img width="308" alt="Screenshot 2023-03-29 at 10 18 17 AM" src="https://user-images.githubusercontent.com/121325991/228618247-39104de0-7a98-44b6-ac10-608c78cf99db.png">
Here's an updated screenshot, am looking into how to create a screenshot gif for future reference